recaptcha_enterprise/snippets/src/main/java/recaptcha/CreateAssessment.java

```diff
@@ -58,7 +64,7 @@ public static void createAssessment(
     try (RecaptchaEnterpriseServiceClient client = RecaptchaEnterpriseServiceClient.create()) {
 
       // Set the properties of the event to be tracked.
-      Event event = Event.newBuilder().setSiteKey(recaptchaSiteKey).setToken(token).build();
+      Event event = Event.newBuilder().setSiteKey(recaptchaSiteKey).setToken(token).setUserIpAddress(userIpAddress).setJa3(ja3).setUserAgent(userAgent).build();
```

Comment:
Kindly chain the commands in separate lines.
```suggestion
      Event event = Event.newBuilder()
          .setSiteKey(recaptchaSiteKey)
          .setToken(token)
          .setUserIpAddress(userIpAddress)
          .setJa3(ja3)
          .setUserAgent(userAgent)
          .build();
```